neptune/management/internal/api.py

```diff
@@ -110,11 +110,13 @@ def get_project_list(api_token: Optional[str] = None) -> List[str]:
     ]
 
 
-def _get_projects_keys_in_organization(workspace: str, api_token: Optional[str] = None) -> Set[str]:
+def _get_projects_keys_in_organization(
+    workspace_id: str, api_token: Optional[str] = None
```

Comment:
I think we can use name instead of id in this API call and avoid listing organizations.
But in this case it will be actually necessary to catch `HTTPNotFound`.

Reply:
retrieval of id was already done in the workspace name validation, so I used it. Then we don't have to secure our code from HTTPNotFound. Ok it is possible, but what is probability of this situation? Moreover workspace_id is used to create new project